prowler/providers/aws/services/workspaces/workspaces_vpc_2private_1public_subnets_nat/workspaces_vpc_2private_1public_subnets_nat.py

```diff
@@ -0,0 +1,53 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.vpc.vpc_client import vpc_client
+from prowler.providers.aws.services.workspaces.workspaces_client import (
+    workspaces_client,
+)
+
+
+class workspaces_vpc_2private_1public_subnets_nat(Check):
+    def execute(self):
+        findings = []
+        for workspace in workspaces_client.workspaces:
+            report = Check_Report_AWS(self.metadata())
+            report.region = workspace.region
+            report.resource_id = workspace.id
+            report.resource_arn = workspace.arn
+            report.resource_tags = workspace.tags
+            report.status = "PASS"
+            report.status_extended = f"Worspace {workspace.id} it's in a VPC wich has 1 public with nat gateway and 2 private subnets"
+            # Find the vpc id for the subnet in the vpc_client
+            vpc_id = False
+            for subnet in vpc_client.vpc_subnets:
+                if subnet.id == workspace.subnet_id:
+                    vpc_id = subnet.vpc_id
+                    break
+            # Get the vpc object from the vpc_client
+            vpc_object = False
+            for vpc in vpc_client.vpcs:
+                if vpc.id == vpc_id:
+                    vpc_object = vpc
+                    break
```

Comment:
I think that maybe is too much but could you change the VPC service to store the `vpc` items in a map instead of a list? The reason behind that is just to perform a quick search instead of looping through all the items of the available VPCs

Reply:
Review @jfagoagas